src/advanced_agent/database/migrations.py

```
@@ -0,0 +1,428 @@
+                        id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        version VARCHAR(50) NOT NULL UNIQUE,
+                        description TEXT,
+                        applied_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+                        checksum VARCHAR(64),
+                        rollback_sql TEXT
+                    )
+                """))
+                session.commit()
+                logger.info("マイグレーション履歴テーブル作成完了")
+        except Exception as e:
+            logger.error(f"マイグレーション履歴テーブル作成エラー: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)